.gitignore

```diff
@@ -1,2 +1,3 @@
 /vendor/
 composer.lock
+/.devcontainer
```

Comment:
Not a blocker but I think this is specific to your use IDE, suggest removing and covering with a global gitignore